eo-maven-plugin/src/main/java/org/eolang/maven/CleanMojo.java

```
@@ -0,0 +1,101 @@
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.eolang.maven;
+
+import com.jcabi.log.Logger;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import org.apache.maven.plugins.annotations.LifecyclePhase;
+import org.apache.maven.plugins.annotations.Mojo;
+
+/**
+ * Implementation of maven clean plugin,
+ * just deleting target/eo directory.
+ *
+ * @since 0.28.6
+ */
+@Mojo(
+    name = "clean",
+    defaultPhase = LifecyclePhase.CLEAN,
+    threadSafe = true
+)
+public class CleanMojo extends SafeMojo {
+
+    @Override
+    final void exec() throws IOException {
+        if (!this.targetDir.exists()) {
+            this.log("Directory %s isn't exist.", targetDir);
+            return;
+        }
+        new Walk(this.targetDir.toPath())
+            .reversed()
+            .stream()
+            .map(Path::toFile)
+            .forEach(
+                dir -> {
+                    this.purge(dir);
+                    this.log("purged %s", dir);
+                }
+            );
+        Logger.info(
+            this,
+            "Deleted all files in: %s",
+            this.targetDir
+        );
+    }
+
+    /**
+     * Logging.
+     *
+     * @param msg Message for logging
+     * @param dir The directory
+     */
+    private void log(final String msg, final File dir) {
+        Logger.info(
+            this,
+            msg,
+            dir.toString()
+        );
+    }
+
+    /**
+     * Remove single file if existed.
+     *
+     * @param file File to purge
+     */
+    private void purge(final File file) {
+        try {
+            Files.deleteIfExists(file.toPath());
+        } catch (final IOException ex) {
+            Logger.error(
```

@l3r8yJ This is called exception swallowing, we are trying to avoid it. Please, read this: https://www.yegor256.com/2015/07/09/catch-if-cant-otherwise.html

---

done